WebDriverAgentLib/Utilities/FBMathUtils.m

```diff
@@ -52,3 +52,13 @@ CGPoint FBInvertPointForApplication(CGPoint point, CGSize screenSize, UIInterfac
       return CGPointMake(screenSize.width - point.y, point.x);
   }
 }
+
+CGSize FBAdjustDimensionsForApplication(CGSize actualSize, UIInterfaceOrientation orientation)
+{
+  if (orientation == UIInterfaceOrientationLandscapeLeft || orientation == UIInterfaceOrientationLandscapeRight) {
+    if (actualSize.height > actualSize.width) {
```

Comment:
XCTest bug is, that it returns exchanged dimensions for landscape mode. This verification is just to make sure the bug is still there (since height is never greater than width in landscape) and to make it still working properly after XCTest itself starts to respect landscape.